src/content/docs/en/guides/cms/decap-cms.mdx

```diff
@@ -8,13 +8,132 @@ i18nReady: true
 ---
 import Grid from '~/components/FluidGrid.astro'
 import Card from '~/components/ShowcaseCard.astro'
+import FileTree from "~/components/FileTree.astro"
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 [Decap CMS](https://www.decapcms.org/) (formerly Netlify CMS) is an open-source, Git-based content management system.
+
+Decap allows you to take full advantage of all of Astro's features, including image optimization and content collections.
+
+Decap adds a route (typically `/admin`) to your project that will load a React app to allow authorized users to manage content directly from the deployed website. Decap will commit changes directly to your Astro project's source repository.
+
+## Installing DecapCMS
+
+There are two options for adding Decap to Astro:
+
+1. [Install Decap via a package manager](https://decapcms.org/docs/install-decap-cms/#installing-with-npm) with the following command:
+
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm install decap-cms-app
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm add decap-cms-app
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add decap-cms-app
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+ 
+ 
+1. Import the package into a `<script>` tag in your page `<body>`
+
+	```html title='/admin'
+    <body>
+      <!-- Include the script that builds the page and powers Decap CMS -->
+      <script src="https://unpkg.com/decap-cms@^3.1.2/dist/decap-cms.js"></script>
+    </body>
+	```
+ 
+
+## Configuration
+
+1. Create a static admin folder at `public/admin/`
+2. Add `config.yml` to `public/admin/`: 
+    <FileTree>
+      - public
+        - admin
+          - config.yml
+    </FileTree>
+3. Configure the `collections` schema in `config.yml`. The following example configures a `blog` collection, defining a `label` for each entry's frontmatter property:
```

Comment:
```suggestion
3. To add support for content collections, configure each `collections` schema in `config.yml`. The following example configures a `blog` collection, defining a `label` for each entry's frontmatter property:
```
Originally, I had an "optional if using content collections" note here, but without that, just saying "the collection schema" didn't really make it clear what "the collection" was.

So, Hope it's OK to add this back in like this!